inductiva/tasks/task.py

```diff
@@ -493,13 +702,27 @@ def get_machine_type(self) -> Optional[str]:
         Returns:
             The machine type, or None if a machine hasn't been assigned yet.
         """
-        info = self.get_info()
-        if info["executer"] is None:
+        info: TaskInfo = self.get_info()
+        if info.executer is None:
             return None
 
-        machine_info = info["executer"]
-
-        machine_provider = machine_info["host_type"]
-        machine_type = machine_info["vm_type"].split("/")[-1]
+        machine_provider = info.executer.host_type
+        machine_type = info.executer.vm_type.split("/")[-1]
 
         return machine_provider + "-" + machine_type
+
+    def print_summary(self, fhandle=sys.stdout):
+        info: TaskInfo = self.get_info()
+
+        # Update running info
+        info.is_running = self.is_running(cached=True)
```

Comment:
We retrieve information from the backend, but then use the cached 'is_running' status, which might be inaccurate. Shouldn't we either obtain both the information and status from the backend or use the cached data for both?

Reply:
For the task info we have:
 - get_info: does an api all and returns the most up to date results
 - info: does not do an api call, and returns the info present in memory
We should do a similar behaviour here. Maybe have a get_summary that always does an api call and a .summary that shows the information present in memory.

Reply:
@sergiomsantos What do you think?

Reply:
Thanks for the suggestions. I reverted the changes of `get_status()`, and added a similar logic inside `TaskInfo` constructor. I also added the `summary` property as suggested by Paulo.